lib/OpenQA/WebAPI.pm

```diff
@@ -81,6 +81,9 @@ sub startup ($self) {
     # set cookie timeout to 48 hours (will be updated on each request)
     $self->app->sessions->default_expiration(48 * 60 * 60);
 
+    my $expire_minion = OpenQA::App->singleton->config->{misc_limits}->{expire_minion_jobs};
+    $self->app->minion->remove_after($expire_minion) if $expire_minion;
```

Comment:
I'm wondering whether this would be better set within the gru service. Likely @kraih knows.

Reply:
Good point, it probably is cleaner to put the code into the `register` method in `OpenQA::Shared::Plugin::Gru`.

Reply:
done